src/mgr/BaseMgrModule.cc

```diff
@@ -461,13 +458,8 @@ ceph_set_module_option(BaseMgrModule *self, PyObject *args)
     derr << "Invalid args!" << dendl;
     return nullptr;
   }
-  auto pModule = PyString_FromString(self->this_module->get_name().c_str());
-  auto pKey = PyString_FromString(key);
-  auto pValue = PyString_FromString(value);
-  auto pArgs = PyTuple_Pack(3, pModule, pKey, pValue);
-  Py_DECREF(pValue);
-  Py_DECREF(pKey);
-  Py_DECREF(pModule);
+  auto pArgs = Py_BuildValue("(ssz)", self->this_module->get_name().c_str(),
```

Comment:
Why did you use `(ssz)`, here. The docs say that `z` is the same as `s`. I'm just wondering if there is any reason for using `z`, here?

Reply:
I used this because the [set_module_option_ex](https://github.com/ceph/ceph/blob/master/src/mgr/BaseMgrModule.cc#L441) parses the string with (ssz).

Reply:
Hmm, ok, I guess it makes more sense to use `z` when parsing since iirc, they do not explicitly state in the docs that `s` accepts `None` when parsing it.